app/components/peopletable/peopletable.js

```diff
@@ -131,6 +143,7 @@ const PeopleTable = translate()(class PeopleTable extends React.Component {
       this.setState({
```

Comment:
Update the test above to:
`if (e === null || _.isEmpty(e.target.value)) {`
To match my previous comment on `handleEmptySearch()`